src/payment/strategies/paypal/paypal-sdk.ts

```diff
@@ -25,6 +31,73 @@ export interface PaypalButtonStyleOptions {
     fundingicons?: boolean;
 }
 
+export interface PaypalActions {
+    payment: PaypalPaymentActions;
+    request: PaypalRequestActions;
+}
+
+export interface PaypalPaymentActions {
+    create(payload: PaypalCreatePaymentPayload): Promise<any>;
```

Comment:
It'd be nice to define the type for the return value instead of `any`.

Reply:
Yeah, I'm not entirely sure what the type it returns even is. Their docs are ...confusing... at best.

Reply:
Yeah I understand. Just try your best. By the way, this is not a blocker.